### ISSUE TITLE
pylint goes silent if improperly configured 

### ISSUE THREAD

**Issue by dciborow** [microsoft/vscode-pylint#297]
I am using a .pylintrc file and have found that incorrect values cause pylint to go silent.

Here is an example pylintrc file that is incorrect because the value is wrapped in quotes.

```
[BASIC]
module-naming-style="any"
```
<img width="1220" alt="image" src="https://user-images.githubusercontent.com/9027725/224389638-e9ba6ce1-7f5d-4435-a876-23cf735b07a2.png">

Once I remove the quotes, pylint begins working again.
```
[BASIC]
module-naming-style=any
```
<img width="1494" alt="image" src="https://user-images.githubusercontent.com/9027725/224389707-a73c04a7-8514-4c65-9d90-bf7cea4b8ed2.png">

It would be nice if a item in the PROBLEMS tab was reported if I have enabled pylint to run but is provided a configuration that causes it to fail.

**Comment by karthiknadig** [comment]
@dciborow Do you see anything in the Output > pylint logs. If you do see something there then we could possibly use that info to create a record in the Problems window.

**Comment by dciborow** [comment]
@karthiknadig ,

Here is the error log.
```
2023-03-12 00:24:45.617 [info] usage: pylint [options]
pylint: error: argument --module-naming-style: invalid choice: '"snake_case"' (choose from 'snake_case', 'camelCase', 'PascalCase', 'UPPER_CASE', 'any')
```

**Comment by bersbersbers** [comment]
This silent failure is a long-standing problem of `pylint` integration in VS Code, which has already lead to misinformation on how to silence specific `pylint` warnings, namely, by means that silenty break `pylint`. Just compare `"--generate-members"` (which does not exist as a `pylint` option) in https://stackoverflow.com/a/59675722/880783, which as collected 40 net upvotes so far.
